test/unit-tests/ad_manager_utils_test.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Unit test class for the Vast Ad Manager
+ * https://github.com/Automattic/expect.js
+ */
+
+var adManagerUtils = require("../../utils/ad_manager_utils.js");
+
+describe('Ad Manager Utility Class:', function()
+{
+
+  it('Time Stamps Should Be Converted to Milliseconds', function()
+  {
+    var milliseconds;
+
+    milliseconds = adManagerUtils.convertTimeStampToMilliseconds("00:00:00");
```

Comment:
Added check for negative numbers. Any argument without 2 ":"'s will not be accepted. As for large numbers, that would be a limitation of the language specification, which in Javascript's case is: 9007199254740991